components/kubearchive/base/kustomization.yaml

```diff
@@ -132,6 +132,9 @@ patches:
                 runAsNonRoot: true
               ports:
                 - containerPort: 8081
+              resources:
+                limits:
+                  memory: 256Mi
```

Comment:
I think we should increase the request as well.

Reply:
this is correct, requests is the most important one in fact. Scheduling is done base on requests and only request is guaranteed. I suggest to use the pattern of setting memory requests=memory limits

Reply:
Changed to make it guaranteed at 256Mi